tensorflow/tools/ci_build/Dockerfile.cpu.arm64

```diff
@@ -1,46 +1,23 @@
-FROM quay.io/pypa/manylinux2014_aarch64
-
-RUN yum -y check-update || true && \
-    yum install -y \
-        sudo \
-        wget \
-        openssl-devel \
-        libffi-devel \
-        java-1.8.0-openjdk-devel \
-        bzip2-devel \
-        gdbm-devel \
-        ncurses-devel \
-        nss-devel \
-        readline-devel \
-        sqlite-devel \
-        epel-release && \
-    yum install -y hdf5-devel && \
-    yum clean all
-
-COPY install/install_bazel.sh /install/
-RUN /install/install_bazel.sh
+FROM linaro/tensorflow-arm64-build:2.12-multipython
 
 ARG py_major_minor_version='3.10'
 
 ENV TF_PYTHON_VERSION=python${py_major_minor_version}
-ENV PYTHON_BIN_PATH=/usr/local/bin/${TF_PYTHON_VERSION}
+ENV PYTHON_BIN_PATH=/usr/bin/${TF_PYTHON_VERSION}
 
 RUN ln -s ${PYTHON_BIN_PATH} /usr/local/bin/python && \
     ln -s ${PYTHON_BIN_PATH} /usr/local/bin/python3
 
-RUN curl -o /tmp/get-pip.py https://bootstrap.pypa.io/get-pip.py && \
-    python /tmp/get-pip.py && \
-    rm -f /tmp/get-pip.py
-
-RUN export PYTHON_VERSION=$(python -c 'import platform; print(platform.python_version())') && \
-    ln -s /opt/_internal/cpython-$PYTHON_VERSION/bin/pip3 /usr/local/bin/pip${py_major_minor_version} && \
-    ln -s /opt/_internal/cpython-$PYTHON_VERSION/bin/pip3 /usr/local/bin/pip3 && \
-    ln -s /opt/_internal/cpython-$PYTHON_VERSION/bin/pip /usr/local/bin/pip
-
-RUN pip3 install packaging
+RUN ${PYTHON_BIN_PATH} -m pip install packaging
 
 ARG is_nightly=0
 ARG tf_project_name='tf_ci_cpu' # PyPI project name passed by CD GitHub workflow
 
 ENV IS_NIGHTLY=${is_nightly}
 ENV TF_PROJECT_NAME=${tf_project_name}
+
+# Set up the master bazelrc configuration file.
+COPY install/.bazelrc /etc/bazel.bazelrc
+
+RUN mkdir /tensorflow
+RUN chmod 777 /tensorflow
```

Comment:
It seems that the Copybara internal presubmit failed because we are not allowed to use `chmod 777` in our files. Do you think we can remove usage of `chmod 777` in this PR?

Reply:
Yes, should be able to.

Reply:
Thanks!